app/code/Magento/Bundle/Helper/Data.php

```diff
@@ -40,4 +50,20 @@ public function getAllowedSelectionTypes()
 
         return isset($configData['allowed_selection_types']) ? $configData['allowed_selection_types'] : [];
     }
+
+    /**
+     * @param $productId
+     *
+     * @return bool
+     */
+    public function isProductQtyDecimal($productId)
```

Comment:
Helper classes are considered to be an anti-pattern. It's best to extract this part in to a separate class with a proper single responsibility. 

Additionally, please consider the following:
* Type hinting for method arguments and return types are required.
* `$productId` variable name hints that it's a product, but the interface contract used below, seems to be a stock item id